core/src/main/java/cf/dmms/app/core/book/BookService.java

```diff
@@ -0,0 +1,31 @@
+package cf.dmms.app.core.book;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class BookService {
```

Comment:
No way to update book?

Reply:
I decided to leave it for now until we know how we want to handle entity changes